src/styles/blocks/card.scss

```diff
@@ -0,0 +1,107 @@
+.card {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  font-style: normal;
+  color: $main-text-color;
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background-color: $card-main-color;
+  transition: transform 0.3s;
+
+  &__title {
+    display: flex;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    text-align: left;
+    margin-bottom: 4px;
+    transition: color 0.3s;
+  }
+
+  &:hover {
+    transform: scale(1.2);
+
+    & .card__title {
+      color: #34568b;
+    }
+  }
+
+  &__image {
+    display: flex;
+    margin-bottom: 40px;
+
+    img {
+      height: 134px;
+      width: 160px;
+      margin: 0 auto;
+    }
+  }
+
+  &__code {
+    font-weight: 400;
+    line-height: 14px;
+    text-align: left;
+    color: $secondary-text-color;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 16px 0 24px;
+
+    .rating-reviews {
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    line-height: 18px;
+
+    .price-text {
+      font-size: 12px;
+      color: $secondary-text-color;
+    }
+
+    .price-value {
+      font-weight: 700;
+      font-size: 16px;
+    }
+  }
+
+  &__buy-button {
+    box-sizing: border-box;
+    text-transform: uppercase;
+    margin: 16px auto;
+    width: 166px;
+    height: 40px;
+    background-color: $nav-text-color-active;
+    padding: 0;
+    border: 0;
+    border-radius: 5px;
+    display: block;
+    color: $card-main-color;
+    font-family: inherit;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    cursor: pointer;
+    transition:
+      color,
+      background-color 0.3s;
```

Comment:
The transition property is split across multiple lines. While this is not incorrect, it could be more concise by combining the properties into a single line, e.g., `transition: color 0.3s, background-color 0.3s;`.